src/index.html

```diff
@@ -15,37 +15,37 @@
     <div class="container">
 
       <div class="calendar calendar__month--length--29">
-        <span class="calendar__day day day--1 day__start--mon"></span>
-        <span class="calendar__day day day--2"></span>
-        <span class="calendar__day day day--3"></span>
-        <span class="calendar__day day day--4"></span>
-        <span class="calendar__day day day--5"></span>
-        <span class="calendar__day day day--6"></span>
-        <span class="calendar__day day day--7"></span>
-        <span class="calendar__day day day--8"></span>
-        <span class="calendar__day day day--9"></span>
-        <span class="calendar__day day day--10"></span>
-        <span class="calendar__day day day--11"></span>
-        <span class="calendar__day day day--12"></span>
-        <span class="calendar__day day day--13"></span>
-        <span class="calendar__day day day--14"></span>
-        <span class="calendar__day day day--15"></span>
-        <span class="calendar__day day day--16"></span>
-        <span class="calendar__day day day--17"></span>
-        <span class="calendar__day day day--18"></span>
-        <span class="calendar__day day day--19"></span>
-        <span class="calendar__day day day--20"></span>
-        <span class="calendar__day day day--21"></span>
-        <span class="calendar__day day day--22"></span>
-        <span class="calendar__day day day--23"></span>
-        <span class="calendar__day day day--24"></span>
-        <span class="calendar__day day day--25"></span>
-        <span class="calendar__day day day--26"></span>
-        <span class="calendar__day day day--27"></span>
-        <span class="calendar__day day day--28"></span>
-        <span class="calendar__day day day--29"></span>
-        <span class="calendar__day day day--30"></span>
-        <span class="calendar__day day day--31"></span>
+        <div class="calendar__day day day--1 calendar__start-day--mon"></div>
```

Comment:
You still write `calendar__start-day--mon` modificator for `day` element
It should be modificator for calendar - the same as `calendar__month--length--29`